client/my-sites/earn/home.tsx

```diff
@@ -542,7 +543,12 @@ const Home = () => {
 					/>
 				</div>
 			) }
-			{ ! isLoading && <PromoSection { ...promos } /> }
+			{ ! isLoading && (
```

Comment:
A good follow-up PR will be to start showing the known shape of these cards as a loading indicator.

Reply:
Yup, I can make a note to review and update this and other Earn page loading indicators.